packages/victory-chart/src/helper-methods.js

```diff
@@ -36,6 +36,37 @@ function getAxisProps(child, props, calculatedProps) {
   };
 }
 
+function getBackgroundWithProps(props, calculatedProps) {
+  const backgroundElement = props.backgroundComponent;
+
+  const height = props.polar
+    ? calculatedProps.range.y[1]
+    : calculatedProps.range.y[0] - calculatedProps.range.y[1];
+  const width = calculatedProps.range.x[1] - calculatedProps.range.x[0];
+
+  const xScale = props.horizontal
+    ? calculatedProps.scale.y.range()[0]
+    : calculatedProps.scale.x.range()[0];
+  const yScale = props.horizontal
+    ? calculatedProps.scale.x.range()[1]
+    : calculatedProps.scale.y.range()[1];
+
+  const xCoordinate = props.polar ? calculatedProps.origin.x : xScale;
+  const yCoordinate = props.polar ? calculatedProps.origin.y : yScale;
+
+  const backgroundProps = {
+    height,
+    polar: props.polar,
+    scale: calculatedProps.scale,
+    style: props.style.background,
+    x: xCoordinate,
+    y: yCoordinate,
+    width
+  };
+
+  return React.cloneElement(backgroundElement, backgroundProps);
```

Comment:
I think we should use lodash `defaults` so that props set directly on the `backgroundComponent` are not nuked by props from the parent.

Like imagine the following scenario where a user wants the background to span the whole height of the svg but be confined to the chart width:

```
<VictoryChart
  height={300}
  width={300}
  style={{ background: { fill: "pink" } }}
  backgroundComponent={<Background y={0} height={300} />
 ...
```
  The `y` value and `height` directly on `Background` should take prececence over what `VictoryChart` is trying to set. 

I think the props should be `defaults({}, backgroundElement.props, backgroundProps)`

Reply:
Oh great point